src/inausoft.netCLI/netCLI.cs

```diff
@@ -0,0 +1,99 @@
+﻿using inausoft.netCLI.Commands;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace inausoft.netCLI
+{
+    public static class netCLI
```

Comment:
I think that class should not starts with lowercase, even the procjest is named netCLI

Reply:
I'm not sure. Perhaps I will adjust the naming if that's something that make you both unconfortable.

Reply:
Adjusted to `Executor.`